android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OnMethodCallMapper.kt

```diff
@@ -34,7 +35,9 @@ class OnMethodCallMapper(private var context: Context, private val oneginiMethod
         } else if (client != null) {
             onSDKMethodCall(call, client, result)
         } else {
-            result.error(OneginiWrapperErrors.ONEGINI_SDK_NOT_INITIALIZED.code, OneginiWrapperErrors.ONEGINI_SDK_NOT_INITIALIZED.message, null)
+            SdkError(
+                wrapperError = OneWelcomeWrapperErrors.ONEWELCOME_SDK_NOT_INITIALIZED
+            ).flutterError(result)
         }
     }
 
```

Comment:
done and TODO

Reply:
looking back, importing each individual constant within this file does not really make sense, so will leave the Constant.event in there

Reply:
If you don't want to import all the constants, you seperate the event constants into a different file and then import all of those. But with this wrapper there are not many constants in the first place so I think you can just import them all.

You would have to rewrite the constants class to be like this. 

```kotlin
object Constants {
```

instead of the current 

```kotlin
interface Constants {
    companion object {
```

Reply:
Objects can not import on the mand resulting in a scenario again where its not import the object with .* forcing me to inforce to import everything manually. In this case i think the Constants prefix is cleaner